go/test/endtoend/cluster/vtctldclient_process.go

```diff
@@ -93,6 +93,18 @@ func VtctldClientProcessInstance(hostname string, grpcPort int, tmpDirectory str
 	return vtctldclient
 }
 
+// PlannedReparentShard executes vtctlclient command to make specified tablet the primary for the shard.
+func (vtctldclient *VtctldClientProcess) PlannedReparentShard(Keyspace string, Shard string, alias string) (err error) {
+	output, err := vtctldclient.ExecuteCommandWithOutput(
+		"PlannedReparentShard",
+		fmt.Sprintf("%s/%s", Keyspace, Shard),
+		"--new-primary", alias)
+	if err != nil {
+		log.Errorf("error in PlannedReparentShard output %s, err %s", output, err.Error())
```

Comment:
there are other places as well we are doing this call, like `InitializeShard` can it be reused

Reply:
We could, but eventually we want to move to using `vtctldclient` and not `vtctlclient`. So I thought it prudent to add a PRS call to vtctldclient. This is what we should be using going forward in all our tests.